python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -760,13 +768,7 @@ def job_finalisation(job, updated_dirac_status):
                     job.updateStatus('running')
                 if job.status in ['completed', 'killed', 'removed']:
                     break
-                # make sure proxy is valid
-                if DiracBase.checkDiracProxy():
-                    # perform finalisation
-                    DiracBase._internal_job_finalisation(job, updated_dirac_status)
-                else:
-                    # exit gracefully
-                    logger.warning("Cannot process job: %s. DIRAC monitoring has been disabled. To activate your grid proxy type: \'gridProxy.renew()\'" % job.fqid)
+                DiracBase._internal_job_finalisation(job, updated_dirac_status)
```

Comment:
The idea is that the check will be preformed at a higher level (in `updateMonitoringInformation()`) so that if the proxy is not present or invalid then it never gets this far.